### PR TITLE
Added reference guide for sensuctl.

### PR DESCRIPTION
**Pull request by mercul3s**
Updated sensuctl install guide to match current version, and moved some
content from there into the ref guide.

### CODE REVIEW

File: content/sensu-core/2.0/reference/sensuctl.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  -t, --timeout string               timeout, in seconds, at which the check has to run
+      --ttl string                   time to live in seconds for which a check result is valid
+{{< /highlight >}}
+
+## Output
+
+sensuctl can be configured to return JSON instead of the default human-readable
+format.
+
+> sensuctl check info marketing-site --format json
```
Comment (palourde) [inline content/sensu-core/2.0/reference/sensuctl.md]: It looks like this format transforms the two consecutive hyphens `--` by an en dash `–` once generated by hugo, so when pasting and running that command:

```bash
$ sensuctl check info marketing-site –format json
[...]
Error: invalid argument(s) received
```

**Review by palourde — APPROVED** [review]
Just a small problem, otherwise 🎉 🎉 🎉 

On another note, I feel like the getting started guide for sensuctl might contain too much information and the installation section should use the platform blocks. That being said, I've submitted https://github.com/sensu/sensu-docs/pull/307 which will make it easier to do that so let's have your PR merged and we can refactor the guide after!

File: content/sensu-core/2.0/reference/sensuctl.md (hunk 1 above)
Comment (mercul3s) [reply content/sensu-core/2.0/reference/sensuctl.md]: Whoops, that to be wrapped in a highlight block!

**Comment by mercul3s** [comment]
@palourde fixed the highlight block. Agreed on refactoring to include platform blocks and trimming it down a bit!

**Comment by palourde** [comment]
LGTM 👍

**Pull request merged**